### PR TITLE
Move some stuff to components.

### PR DESCRIPTION
**Pull request by brantje**

### CODE REVIEW

File: src/components/DownloadCard.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+    @Component({components: {}})
+    export default class ButtonCard extends Vue {
+        @Prop(String) public address!: string;
+    }
+</script>
+<style scoped>
+    .button-card {
+        position: relative;
```
Comment (danimoh) [inline src/components/DownloadCard.vue]: No need for `position: relative` here

File: src/components/DownloadCard.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+    .nq-h1 {
+        text-align: center;
+    }
+    .nq-text{
+        text-align: center;
+        padding: 0 17px;
```
Comment (danimoh) [inline src/components/DownloadCard.vue]: Use `max-width` over `padding` here too.

File: src/components/DownloadCard.vue [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    .qr-code {
+        height: 178px;
+        width: 178px;
+        background-color: red;
+        margin: 0 auto 40px;
+    }
+    .nq-text.link {
+        font-family: Fira Mono;
+        font-style: normal;
+        font-weight: normal;color: #1F2348;
```
Comment (danimoh) [inline src/components/DownloadCard.vue]: Missing new line.

Also use `var(--nimiq-blue)`

File: src/components/RecipientCard.vue [hunk 4]
```diff
@@ -0,0 +1,54 @@
+<template>
+    <SmallPage class="recipient-card">
+        <Identicon :address="address"></Identicon>
+        <AddressDisplay :address="address"></AddressDisplay>
+        <div class="seperator"></div>
```
Comment (danimoh) [inline src/components/RecipientCard.vue]: Misspelled: `separator`
Semantically better would be to use a `hr` and style it accordingly.

File: src/components/RecipientCard.vue [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        <input class="nq-input vanishing" type="text" placeholder="Add a Message"/>
+        <div class="nq-text-s">(optional)</div>
+    </SmallPage>
```
Comment (danimoh) [inline src/components/RecipientCard.vue]: The first UI state with the address input and the login button is missing.

File: src/components/RecipientCard.vue [hunk 6]
```diff
@@ -0,0 +1,54 @@
+<template>
+    <SmallPage class="recipient-card">
+        <Identicon :address="address"></Identicon>
+        <AddressDisplay :address="address"></AddressDisplay>
+        <div class="seperator"></div>
+        <input class="nq-input vanishing" type="text" placeholder="Add a Message"/>
```
Comment (danimoh) [inline src/components/RecipientCard.vue]: Use the message.

File: src/components/RecipientCard.vue [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+        <div class="nq-text-s">(optional)</div>
+    </SmallPage>
+</template>
+
+
+<script lang="ts">
+    import '@nimiq/style/nimiq-style.min.css';
```
Comment (danimoh) [inline src/components/RecipientCard.vue]: We already import this in `App.vue`.
That's enough.
We don't specifically repeat this for each component.

File: src/components/RecipientCard.vue [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+    .recipient-card .address-display {
+        margin-bottom: 39px;
+    }
+
+    .recipient-card .seperator {
+        height: 1px;
+        width: 100%;
+        opacity: 0.1;
+        border: 1px solid #1F2348;
```
Comment (danimoh) [inline src/components/RecipientCard.vue]: use `var(--nimiq-blue)`

File: src/components/RecipientCard.vue [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+        margin-bottom: 30px;
+    }
+
+    .nq-input.vanishing {
+        text-align: center;
+    }
+
+    .recipient-card .nq-text-s {
+        color: #1F2348;
```
Comment (danimoh) [inline src/components/RecipientCard.vue]: Use `var(--nimiq-blue)

File: src/components/RecipientCard.vue [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+    .recipient-card .seperator {
+        height: 1px;
+        width: 100%;
+        opacity: 0.1;
+        border: 1px solid #1F2348;
+        margin-bottom: 30px;
+    }
+
+    .nq-input.vanishing {
```
Comment (danimoh) [inline src/components/RecipientCard.vue]: What is this 'vanishing` supposed to be?

Comment (sisou) [reply src/components/RecipientCard.vue]: It's a style framework modifier class to make the input borders disappear.

File: src/components/RecipientCard.vue [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+        text-align: center;
+    }
+
+    .recipient-card .nq-text-s {
+        color: #1F2348;
+        /* Dark Blue */
+        opacity: 0.3;
+    }
+</style>
```
Comment (sisou) [inline src/components/RecipientCard.vue]: Please configure your editor to add newlines at the end of files.

**Comment by danimoh** [comment]
These changes have been merged as part of https://github.com/nimiq/donation/pull/3

**Pull request closed without merging**